recipes/databroker/v0.3.0/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: databroker
+  version: v0.3.0
+
+source:
+  git_url: https://github.com/NSLS-II/databroker
+  git_rev: master
```

Comment:
This should be pinned to a tag, not to master.  I now understand why I was confused.